src/component/SearchPage/SearchPage.js

```diff
@@ -19,10 +32,33 @@ const SearchPage = () => {
                 Search for Tutors
                 </h1>
             <div className="search3">
-                Not available right now
-                </div>
-
+                {tutors.map(tutor => {
+                    return (
+                        <div key={tutor._id}>
+                            {tutor.subject}<br />
+                            {tutor.description}<br />
+                            {`${tutor.user.firstName} ${tutor.user.lastName}`}<br />
+                            <a onClick={() => {
+                                setShowModal(true)
+                                setCurrentSubjectDescription(tutor.description)
+                                setCurrentTutorId(tutor.user._id)
+                                setCurrentSubjectTitle(tutor.subject)
+                                setCurrentTutorName(`${tutor.user.firstName} ${tutor.user.lastName}`)
+                            }}>Check Availability</a>
+                        </div>
+                    )
+                })}
+            </div>
 
+            {<ModalWindow
+                modalShow={showModal}
+                hideModal={hideModal}
+                subjectDescription={currentSubjectDescription}
+                tutorId={currentTutorId}
+                subjectTitle={currentSubjectTitle}
+                tutorName={currentTutorName}
+                studentId={localStorage.token}
```

Comment:
I think the current version of the route accepts a userId, it doesn't accept a token.  The POST to /meetings/create returns a status of 500 when the token is used.